app/helpers/remote_execution_helper.rb

```diff
@@ -34,7 +34,7 @@ def template_invocation_actions(task, host, job_invocation, template_invocation)
         action: { href: current_host_details_path(host), 'data-method': 'get', id: "#{host.name}-actions-detail" } }
     end
     if authorized_for(controller: :job_invocations, action: :create) && (!host.infrastructure_host? || User.current.can?(:execute_jobs_on_infrastructure_hosts))
-      links << { title: (_('Rerun on %s') % host.name),
+      links << { title: (_('Rerun on %s') % host),
```

Comment:
Is the result of `host` and `host.to_label` same?
If yes, we should unify the usage, and use it same way across the code.

Reply:
You are right. I'll change it where possible.